Sites landing page (#634)
edgio-docs/edgio-docs: src/pages/guides/sites_frameworks.md

title: Sites (Frameworks)
---

{{ PRODUCT }} {{ PRODUCT_PLATFORM }}, which provides optimal performance for your headless Jamstack applications, is powered by [{{ PRODUCT }} {{ PRODUCT_EDGE }}](/guides/performance). This means that you will be able to develop your application at a faster rate and it empowers you to define performance optimizations within your application code. Build upon these performance improvements through static page generation:
-   At build time through Static Site Generation (SSG).
-   After your site has been built through [Incremental Static Regeneration (ISR)](/guides/sites_frameworks/isg).

> -   When a client requests a page through Server-Side Rendering (SSR). SSR generates static pages through our Serverless Compute workers. In addition to either eliminating the need for web servers or reducing their load, this improves performance by processing requests closer to your users.

Reviewer: We should change this to focus more on SSR rather than SSG. Since other platforms were built to cater to SSG use cases and are relatively poor at SSR use cases and our platform is stronger at SSR comparatively.
In general, SSR is preferred by users over SSG because it offers similar time-to-first-byte performance through caching but is inherently dynamic.

> SSR generates static pages through our Serverless Compute workers. In addition to either eliminating the need for web servers or reducing their load, this improves performance by processing requests closer to your users.

I think we need to rephrase the above, SSR doesn't really generate static pages, that's ISR. And replaces web servers, doesn't reduce their load. The SSR requests are not processed closer to the users necessarily. They can be cached closer to the users but that's the case for SSG and ISR as well.

## Getting Started

> {{ PRODUCT }} {{ PRODUCT_PLATFORM }} provides over [40 popular frameworks integrations](/guides/sites_frameworks/getting_started) through which you can deploy your headless Jamstack applications to {{ PRODUCT }}. Getting started with {{ PRODUCT }} {{ PRODUCT_PLATFORM }} involves initializing your {{ PRODUCT }} property through our CLI. Our CLI will then automatically detect your framework and configure your app accordingly.

Reviewer: > you can deploy your headless Jamstack applications

Let's rephrase this to "websites and apps" or "webapps" or something similar. We don't want to give the impression that our platform enables you to develop mobile/native applications.

Reviewer: > property

Are we sure we want to call this property? If so, then it's okay. But let's check with the console team on what they'll be calling it. I know that it's currently called "site".

<Callout type="tip">

  If you are still planning or developing your headless Jamstack application, then consider using our [Traffic Splitting](/guides/performance/traffic_splitting) capability to migrate to headless Jamstack in stages.

</Callout>

## Regions {/*regions*/}

> {{ PRODUCT }} {{ PRODUCT_PLATFORM }} is available in the following global regions:

Reviewer: Let's make sure that the readers understand the these are the regions where the compute is available; the cache will be distributed to 100s of PoPs around the world, closer to end users.

-   **Americas**: Eastern US and Western US
-   **Europe**: Ireland, UK, Western Europe, Northern Europe, Central Europe
-   **Asia**: Japan
-   **Oceania**: Australia

<Callout type="info">

  Enterprise customers may choose the region where their workloads will run. Alternatively, if you are using our free tier, then your workloads will run in the Eastern US region.

</Callout>

Edgio ensures high availability for your computing workloads by:

-   Running your code within two separate data centers.
-   Setting up automatic DNS failover between those data centers.

> -   Load balancing computing requests between redundant processes.

Reviewer: Not sure about this one. I don't think we're doing any load balancing based on usage. @phdunham can you confirm please!

> title: Sites (Frameworks)

Reviewer: ```suggestion
title: Sites
```

---

> {{ PRODUCT }} {{ PRODUCT_PLATFORM }}, which provides optimal performance for your Jamstack sites or applications, is powered by [{{ PRODUCT }} {{ PRODUCT_EDGE }}](/guides/performance). This means that you will be able to develop your application at a faster rate and it empowers you to define performance optimizations within your application code. 

Reviewer: ```suggestion
{{ PRODUCT }} {{ PRODUCT_PLATFORM }} is a Jamstack platform built from the group up to run high-performance websites and web apps. It is the only Jamstack platform with an integrated world-class CDN [{{ PRODUCT }} {{ PRODUCT_EDGE }}](/guides/performance) and state-of-the-art [web security suite](/guides/security). {{ PRODUCT }} {{ PRODUCT_PLATFORM }} enables you to build your web apps faster and leverage a global CDN in the application code to achieve the best performance in production. 
```

> Build upon these performance improvements through Server-Side Rendering (SSR). SSR allows you to compute code embedded within your web application using our Serverless Compute workers. Processing your code  within the {{ PRODUCT }} cloud eliminates the need for web servers to run that code. More importantly, it improves performance  through caching without compromising the dynamic nature of these requests. Responses served from cache will achieve a comparable Time to First Byte (TTFB) as static cached content.

Reviewer: ```suggestion
Build upon these performance improvements through Server-Side Rendering (SSR). {{ PRODUCT }} {{ PRODUCT_PLATFORM }} takes an SSR-first approach and enables you to render your web apps using our Serverless Compute at any scale. This means that you don't have to maintain and scale the backend server infrastructure and your teams can focus on shipping features your customers will love. {{ PRODUCT }} {{ PRODUCT_PLATFORM }} improves real-world performance through [fine-grained caching](/guides/performance/caching#cache-key), allowing you to cache highly dynamic content. Responses served from the CDN cache will achieve a comparable Time to First Byte (TTFB) as static cached content.
```

We also support static page generation:
-   At build time through Static Site Generation (SSG).
-   After your site has been built through [Incremental Static Regeneration (ISR)](/guides/sites_frameworks/isg).

## Getting Started {/*getting-started*/}

{{ PRODUCT }} {{ PRODUCT_PLATFORM }} provides over [40 popular frameworks integrations](/guides/sites_frameworks/getting_started) through which you can deploy your websites and applications to {{ PRODUCT }}. Getting started with {{ PRODUCT }} {{ PRODUCT_PLATFORM }} involves initializing your {{ PRODUCT }} property through our CLI. Our CLI will then automatically detect your framework and configure your app accordingly.

<Callout type="tip">

  If you are still planning or developing your website or application, then consider using our [Traffic Splitting](/guides/performance/traffic_splitting) capability to migrate to a Jamstack site or application in stages.

</Callout>

## Compute Regions {/*regions*/}

> {{ PRODUCT }} allows you to cache and deliver your content through any of our points of presence (POPs) around the world. However, your serverless code will be computed within one of the following global regions:

Reviewer: ```suggestion
{{ PRODUCT }} allows you to cache and deliver your content through any of our points of presence (POPs) around the world. However, your serverless code will run in one of the following global regions:
```

-   At build time through Static Site Generation (SSG).
-   After your site has been built through [Incremental Static Regeneration (ISR)](/guides/sites_frameworks/isg).
-   When a client requests a page through Server-Side Rendering (SSR). SSR generates static pages through our Serverless Compute Lambda workers. This improves performance by processing requests closer to your clients and reduces the load on your web servers.

{{ PRODUCT }} {{ PRODUCT_PLATFORM }} provides over [40 popular frameworks integrations](/guides/sites_frameworks/getting_started) through which you can deploy your headless Jamstack applications to {{ PRODUCT }}. Getting started with {{ PRODUCT }} {{ PRODUCT_PLATFORM }} involves initializing your {{ PRODUCT }} property through our CLI. Our CLI will then automatically detect your framework and configure your app accordingly.

<Callout type="tip">

  If you are still planning or developing your headless Jamstack application, then consider using our [Traffic Splitting](/guides/performance/traffic_splitting) capability to migrate to headless Jamstack in stages.

</Callout>

## Regions {/*regions*/}

> {{ PRODUCT }} is available in the following global regions:

Reviewer: I think this should be `{{ PRODUCT }} {{ PRODUCT_PLATFORM }}`
